### PR TITLE
Monorepo

### PR DESCRIPTION
**Pull request by oveddan**
First step of migrating to a monorepo:
* 1155 contracts moved into a yarn workspace, at `packages/1155-contracts`
* Added turborepo for the build/test/other ci task pipelines
* Updated github workflows to account for running tasks using turborepo pipelines
* added readme to root

### CODE REVIEW

**Comment by oveddan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #241** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/241" target="_blank">Graphite</a> 
    * **PR #242** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/242" target="_blank">Graphite</a> 
      * **PR #243** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/243" target="_blank">Graphite</a> 
        * **PR #244** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/244" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/244?utm_source=stack-comment).

File: package.json [hunk 1]
```diff
@@ -1,68 +1,39 @@
...
-  "name": "@zoralabs/zora-1155-contracts",
-  "version": "2.0.3",
-  "repository": "git@github.com:ourzora/creator-contracts.git",
-  "author": "Iain <iain@zora.co>",
-  "license": "MIT",
-  "main": "./dist/index.cjs",
-  "types": "./dist/package/index.d.ts",
-  "type": "module",
+  "private": true,
+  "repository": "git@github.com:ourzora/zora-contracts.git",
```
Comment (oveddan) [inline package.json:3]: @iainnash what should the repo url be?

**Comment by kulkarohan** [comment]
thoughts on creating a fresh new repo for the monorepo over merging into this?

**Comment by oveddan** [comment]
i can see the argument for that, but i think it would be ideal if we had all the PR history in this same repo and also existing stars/forks? What do you think would be the advantages to make a new repo?

File: package.json (hunk 1 above)
Comment (oveddan) [reply package.json:3]: lets call it protocol!

File: package.json [hunk 2]
```diff
@@ -1,68 +1,39 @@
...
-    "vitest": "~0.30.1"
+    "tsconfig": "*",
+    "turbo": "latest"
   },
-  "devDependencies": {
-    "@turnkey/api-key-stamper": "^0.1.1",
-    "@turnkey/http": "^1.2.0",
-    "@turnkey/viem": "^0.2.4",
-    "@types/node": "^20.1.2"
+  "name": "turborepo-example",
```
Comment (oveddan) [inline package.json:20]: rename the name here

**Review by IsabellaSmallcombe — APPROVED** [review]

**Comment by oveddan** [comment]
### Merge activity

* **Oct 12, 12:55 PM**: @oveddan started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/244).

**Pull request merged**